ext_emconf.php

```diff
@@ -25,13 +25,13 @@
 	'uploadfolder' => '0',
 	'createDirs' => '',
 	'modify_tables' => '',
-	'clearCacheOnLoad' => 0,
+	'clearCacheOnLoad' => 1,
```

Comment:
I'm not 100% sure. So I've changed this back.

(In general, I have a bad feeling when the cache is not cleared after installing a FE-related extension, but I would still need to spend time on finding a case where this is a problem for this particular extension.)